outputs.md

```diff
@@ -17,64 +17,101 @@ FHD outputs various data products. We outline and describe these below. <br />
-**cal.skymodel.source_list**: an array of structures of dimension N<sub>sources</sub>.
-  * **x**: !Q, **y**: !Q, **ra**: the RA coordinate value of the source, **dec**: the DEC coordinate value of the source, **ston**: not used in the calibration source list (!Q right?), **freq**: frequency of the source model, **alpha**: spectral index of the source, **gain**: not used in the calibration source list (!Q right?), **flag**: type codes where 0 is no flag, 1 is low confidence, and 2 is sidelobe contamination
-  * **extend**: !Q 
-  * **flux**: structure of the fluxes for that source, where the dimension depends on what polarizations are reported for that source. The order of potential polarizations is xx, yy, xy, and yx in apparent brightness or I, Q, U, V in sky brightness.
+**cal**: a structure of the various calibration parameters. Pointers and large arrays in this structure are defined below. Other variables in the structure include:
+  * **n_pol**: long number of polarizations
+  * **n_freq**: long number of frequency channels
+  * **n_tile**: long number of tiles
+  * **n_time**: long number of time steps
+  * **auto_initialize**: integer 1/0 value of whether window scaled autocorrelations were used/not used
+  * **max_iter**: long number of maximum iterations in the X<sup>2</sup> calibration subroutine
+  * **phase_iter**: long number of maximum iterations in the X<sup>2</sup> calibration subroutine where only the phase was updated
+  * **auto_scale**: !Q
+  * **tile_names**: string array of the tile names specified in the metafits and if unspecified then indices of tiles starting at 1
+  * **freq**: float array of frequency values in Hz
+  * **min_cal_baseline**: float of the minimum baseline length included in calibration
+  * **max_cal_baseline**: float of the maximum baseline included in calibration
+  * **n_vis_cal**: long number of visibilities used in calibration
+  * **time_avg**: integer 1/0 value of whether or not visibilities were averaged in time during the X<sup>2</sup> calibration subroutine
+  * **min_sols**: integer minimum number of equations used in the X<sup>2</sup> calibration subroutine per frequency channel
+  * **ref_antenna**: long number index of tile to reference phases from
+  * **ref_antenna_name**: name of the antenna used as the phase reference.
+  * **conv_thresh**: float number of the required convergence threshold
+  * **polyfit**: integer 1/0 value of whether or not polynomials were fit
+  * **amp_degree**: integer number of degree of coefficients used in fitting the amplitude of the cal solutions i.e. 2 = 2<sup>nd</sup> order polynomial
+  * **phase_degree**: integer number of degree of coefficients used in fitting the phase of the cal solutions i.e. 2 = 2<sup>nd</sup> order polynomial
+  * **bandpass**: integer 1/0 value of whether or not a bandpass was fit
+  * **mode_fit**: integer 1/0 value of whether or not cable reflection mode-fitting was done,
+  * **mean_gain**: !Q
+  * **mean_gain_residual**: !Q
```

Comment:
The average amplitude of the residual calibration solutions, after the fitted solutions have been subtracted.